### ISSUE TITLE
On chain transaction fails if Fast fee option is selected.

### ISSUE THREAD

**Issue by pcwd** [LN-Zap/zap-android#341]
When sending an on-chain payment using the Fast fee option, there is no fee estimate displayed, only N/A.  When you click send, you get "ERROR: unable to query fee estimator: conf target of 1 is too low, minimum accepted is 2" and the payment fails to send.

**Comment by michaelWuensch** [comment]
@pcwd 
Thanks for the report.
I actually never used high fees :D
I am wondering though if that was changed in an LND update as no one ever reported this so far.
I will have a look at it and fix it for the next release.

Which version of LND are you using?

**Comment by michaelWuensch** [comment]
Interesting,
I just tested it on my lnd 12.1 node with fast and a fee was calculated without a problem. I didn't send though, as I didn't want to transact anyting.
I have had a look at the lnd code and the min for 12.1 and 13.0 is actually set at 2. So I don't understand, why this works on the 12.1 node.
But they actually now changed the minimum to 1 a few days ago, which was released in lnd 13.1 today. (https://github.com/lightningnetwork/lnd/commit/1abc5f8b0040cb83b431e50811029f2c200ca0da#diff-a8b2bfb868fb8e3cc567762cdc0851314b65e977c06d7d948e1dc2bbedc4299a)
So I think I actually won't change anything.
